src/intents.cs

```diff
@@ -2879,22 +2880,22 @@ interface INImage : NSCopying, NSSecureCoding {
 
 		// INImage_IntentsUI (IntentsUI)
 
-		[NoMac, NoWatch]
+		[NoMac, Watch (5,0)]
```

Comment:
Because this `Watch (5,0)` is by no means true 🤦‍♂️ or at least not yet in Beta 1... the iOS header says so but there is not really changes in the watchOS SDK, nice catch